<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditLayer</class>
 <widget class="QDialog" name="EditLayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>369</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edition de couche</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Type de couche</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="radioButton">
        <property name="text">
         <string>Vidéo complète</string>
        </property>
        <property name="icon">
         <iconset resource="ressources.qrc">
          <normaloff>:/icons/media</normaloff>:/icons/media</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_2">
        <property name="text">
         <string>Plans</string>
        </property>
        <property name="icon">
         <iconset resource="ressources.qrc">
          <normaloff>:/icons/media_shots</normaloff>:/icons/media_shots</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_3">
        <property name="text">
         <string>Images fixes</string>
        </property>
        <property name="icon">
         <iconset resource="ressources.qrc">
          <normaloff>:/icons/media_picture</normaloff>:/icons/media_picture</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Nom de la couche :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="nameEdit">
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string>Animation</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QToolButton" name="browseButton">
       <property name="text">
        <string>Parcourir...</string>
       </property>
       <property name="icon">
        <iconset resource="ressources.qrc">
         <normaloff>:/icons/folder_open</normaloff>:/icons/folder_open</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="pathEdit">
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string>Chemin...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="recursiveButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Recherche récursive
(regarder dans tous les sous-dossiers)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="detailsWidget">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Fichiers</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Nomenclature :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="uiPrefix">
          <property name="frame">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="placeholderText">
           <string>prefix or *</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="uiFileName">
          <property name="frame">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>filename</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="uiSuffix">
          <property name="frame">
           <bool>false</bool>
          </property>
          <property name="placeholderText">
           <string>suffix or *</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="uiExtension">
          <property name="frame">
           <bool>false</bool>
          </property>
          <property name="placeholderText">
           <string>ext</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="seqWidget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QRadioButton" name="seqButton">
           <property name="text">
            <string>Séquences d'images</string>
           </property>
           <property name="icon">
            <iconset resource="ressources.qrc">
             <normaloff>:/icons/media_picture</normaloff>:/icons/media_picture</iconset>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="videoButton">
           <property name="text">
            <string>Fichiers Vidéos</string>
           </property>
           <property name="icon">
            <iconset resource="ressources.qrc">
             <normaloff>:/icons/media</normaloff>:/icons/media</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditLayer</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditLayer</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
